baselines/ppo2/ppo2.py

```diff
@@ -411,4 +431,117 @@ def fn(x):
 def safemean(xs):
     return np.nan if len(xs) == 0 else np.mean(xs)
 
+def eval_and_viz_tom(additional_params, env, model,run_info):
+    """
+    The ppo agent will play with a selection of other agents, to evaluate performance, including TOMs with
+    params 0 and 1 (who they'll also train with), and two different BC agents. They all play with both indices.
+    """
```

Comment:
You might want to move this function to your project repo, and then import the relevant file within your if statement. This doesn't really seem to belong to baselines.